### DISCUSSION TITLE
FastApi has no Attribute debug error

### DISCUSSION THREAD

**Discussion by jayapratha111998** [fastapi/fastapi#9488] [Questions] [answered]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

i have given instance like this : 
![Screenshot (14)](https://user-images.githubusercontent.com/109121102/236450739-9a468b6f-96dc-46f1-8792-66ada5770bcf.png)

app=FastAPI()   
when i run a fastapi file i got Error AttributeError: 'FastAPI' object has no attribute 'debug' like this,
 how can i solve this
![Screenshot (8)](https://user-images.githubusercontent.com/109121102/236450858-30967569-e10d-42e9-a186-b585728c279b.png)

**Comment by Kludex** [comment, marked as answer]
Bump your FastAPI version.

**Reply by jayapratha111998** [reply]
thank u
